yellow-paper/docs/contracts/index.md

```diff
@@ -12,6 +13,138 @@ The purpose of the L1 contracts are simple:
 - Act as a validating light node for L2 that every L1 node implicitly run
 :::
 
+## Overview
+
+When presented with a new `ProvenBlock` and its proof, an Aztec node can be convinced of its validity if the proof passes and the `Header.last_archive` matches the `archive` of the node (archive here represents a root of [archive tree](../state/archive.md)). The `archive` used as public input is the archive after the new header is inserted (see [root rollup](./../rollup-circuits/root_rollup.md)).
```

Comment:
Perhaps include a link to where `ProvenBlock` is defined.
Edit: same comment for when other structs in this page are mentioned for the first time